### PR TITLE
Summary processing

### PR DESCRIPTION
**Pull request by mcyprian**
- shorten summary if too long
- remove quotes and whitespaces
- fix unwanted removing of single dashes from description and summary

### CODE REVIEW

**Review by hroncok — COMMENTED** [review]
I'd make it so for summary, to cut it before first newline unconditionally, multiline summaries are bogus anyway.

File: pyp2rpm/metadata_extractors.py [hunk 1]
```diff
@@ -28,6 +28,22 @@
...
+    """Shorten given text on first delimiter after given number
+    of characters.
+    """
+    if not isinstance(delims, list):
+        delims = [delims]
+
+    for delim in delims:
+        cut = text.find(delim, length)
+        if cut != -1:
+            break
```
Comment (hroncok) [inline pyp2rpm/metadata_extractors.py]: This does not cut it after first delimiter. See:

```python
>>> cut_to_length("foo bar. zap: foo", 6, ['.', ':'])
'foo bar'
>>> cut_to_length("foo bar. zap: foo", 6, [':', '.'])
'foo bar. zap'

```

Comment (mcyprian) [reply pyp2rpm/metadata_extractors.py]: Is the issue that docstring doesn't match the implementation or the implementation itself? Only one delimiter is always used as an argument at this moment.

Comment (hroncok) [reply pyp2rpm/metadata_extractors.py]: Well if the current behavior is desired, update the docstring.

**Comment by mcyprian** [comment]
I changed the implementation of cut_to_length function to be as simple as possible for now.

**Comment by mcyprian** [comment]
@hroncok can this be merged?

**Review by hroncok — APPROVED** [review]

**Comment by hroncok** [comment]
Go ahead.

**Pull request merged**